.github/workflows/release-helm.yml

```diff
@@ -120,6 +125,10 @@ jobs:
         id: get_version
         run: ./scripts/check-helm-version.sh
         shell: bash
+      - name: Get Versions
+        run: |
+          echo "##[set-output name=TOOLCHAIN;]$(cat rust-toolchain)"
+          echo "##[set-output name=VERSION;]$(tomlq .package.version node/Cargo.toml | tr -d '"')"
```

Comment:
I think there's some confusion here. This version will currently never be used right because:

```
version: ${{ steps.get_version.outputs.VERSION }}
```

Do you need the `check-version-helm` script to output two versions `HELM_VERSION` and `APP_VERSION`?

Reply:
Yes I just noticed this.